### PR TITLE
So5 bug fix

### PR DESCRIPTION
**Pull request by hiteshdigit101**
ADBP-903 bug fix implemented

### CODE REVIEW

File: src/components/adb-transaction-history/index.tsx [hunk 1]
```diff
@@ -245,38 +249,38 @@ const ADBTransactionHistoryComponent = (prop: ADBTransactionHistoryProps) => {
...
+  console.log("wallets", wallets);
 
   const markedDays: any = {};
 
   if (selectedFromDate) {
     markedDays[selectedFromDate] = {
       startingDay: true,
-      color: '#1B1B1B',
-      textColor: '#FFFFFF',
+      color: "#1B1B1B",
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:260]: plz use template colors

File: src/components/adb-transaction-history/index.tsx [hunk 2]
```diff
@@ -245,38 +249,38 @@ const ADBTransactionHistoryComponent = (prop: ADBTransactionHistoryProps) => {
...
-      m.add(1, 'days')
+      m.add(1, "days")
     ) {
-      markedDays[m.format('YYYY-MM-DD')] = {
+      markedDays[m.format("YYYY-MM-DD")] = {
         startingDay: false,
-        color: '#1B1B1B',
-        textColor: '#FFFFFF',
+        color: "#1B1B1B",
+        textColor: "#FFFFFF",
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:275]: plz use template colors

File: src/components/adb-transaction-history/index.tsx [hunk 3]
```diff
@@ -430,15 +461,15 @@ const ADBTransactionHistoryComponent = (prop: ADBTransactionHistoryProps) => {
                     style={[
                       styles.optionsContainer,
                       selectedFilterDateOptionValue === opt.value && {
-                        backgroundColor: '#1B1B1B',
+                        backgroundColor: "#1B1B1B",
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:464]: plz use template colors

File: src/components/adb-transaction-history/index.tsx [hunk 4]
```diff
@@ -482,35 +513,35 @@ const ADBTransactionHistoryComponent = (prop: ADBTransactionHistoryProps) => {
...
-              <View style={{ flexDirection: 'row', marginTop: 30 }}>
+              <View style={{ flexDirection: "row", marginTop: 30 }}>
                 <View style={{ flex: 1 }}>
                   <ADBButton
-                    label={'Reset'}
-                    background={'#FFFFFF'}
-                    labelColor={'#1B1B1B'}
+                    label={"Reset"}
+                    background={"#FFFFFF"}
+                    labelColor={"#1B1B1B"}
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:533]: plz use template colors

**Review by sharmal33 — CHANGES_REQUESTED** [review]
plz use template colors, don't hardcode the colors

File: src/components/adb-transaction-history/index.tsx (hunk 1 above)
Comment (hiteshdigit101) [reply src/components/adb-transaction-history/index.tsx]: I have changed it to theme colors

File: src/components/adb-transaction-history/index.tsx (hunk 2 above)
Comment (hiteshdigit101) [reply src/components/adb-transaction-history/index.tsx]: I have changed it to theme colors

File: src/components/adb-transaction-history/index.tsx (hunk 3 above)
Comment (hiteshdigit101) [reply src/components/adb-transaction-history/index.tsx]: I have changed it to theme colors

File: src/components/adb-transaction-history/index.tsx (hunk 4 above)
Comment (hiteshdigit101) [reply src/components/adb-transaction-history/index.tsx]: I have changed it to theme colors

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**